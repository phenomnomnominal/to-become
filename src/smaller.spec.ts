import { smaller } from './index';

describe(`toBecome(smaller)`, () => {
    it(`it should work for values that shrink`, () => {
        expect(3133634400000 - Date.now()).toBecome(smaller);
    });

    it(`it should work for values that stay the same size`, () => {
        expect(1000).toBecome(smaller);
    });
});
