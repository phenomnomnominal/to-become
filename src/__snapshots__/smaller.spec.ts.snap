// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toBecome(smaller) it should work for values that shrink 1`] = `1566165687504`;

exports[`toBecome(smaller) it should work for values that stay the same size 1`] = `1000`;
